docs/Guide/CLI/getting-started.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Getting Started
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
+
+# Getting Started
+
+## Installation
+
+<Tabs groupId="packageManagers">
+<TabItem value="yarn" label="Yarn (Recommended)" default>
+
+```shell:no-line-numbers
+$ yarn global add @sapphire/cli
+```
+
+</TabItem>
+
+<TabItem value="npm" label="NPM">
+
+```shell:no-line-numbers
+$ npm install @sapphire/cli -g
+```
+
+</TabItem>
+</Tabs>
```

Comment:
![image](https://user-images.githubusercontent.com/4019718/139582037-91774bdd-28f4-4dc4-a55a-9cb5c5b2154b.png)

codeblocks don't let me post it as code

Reply:
?

Reply:
@Favna evaluate